### PR TITLE
5 portals description

### PR DESCRIPTION
**Pull request by Anna-Pronovich**
Снимок

### CODE REVIEW

**Comment by JuliaMV** [comment]
please, add styles and delete all comments and unnecessary files,

File: src/components/portaldescription/OurTeamStyles.module.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.teamLine_big,
+.teamLine_small {  
+  display: flex;
+  flex-direction: row;
```
Comment (GGWebProject) [inline src/components/portaldescription/OurTeamStyles.module.scss]: default value of felx-direction is row. you don`t change that property and you can del this row

File: src/components/portaldescription/OurTeamStyles.module.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  flex-wrap: wrap;
+  justify-content: space-around;
+  margin: 0 auto;
+}
+
+.teamLine_big {
```
Comment (GGWebProject) [inline src/components/portaldescription/OurTeamStyles.module.scss]: same css selectors for block and modificator:
`teamLine_big, teamMember_avatar, teamMember_imgLink`

small, big is modificators and teamMember_avatar - is block

**Comment by Anna-Pronovich** [comment]
I added new files with changes

File: src/pages/indexStyle.module.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.sectionWrapper {
+  margin: 20px 0 60px;
```
Comment (JuliaMV) [inline src/pages/indexStyle.module.scss]: I suppose don't use a separate file for one rule and rewrite this in inline styles

**Pull request merged**